app/Http/Controllers/EmployeeController.php

```diff
@@ -148,8 +149,14 @@ public function update(Request $request , string $id){
     }
 
     public function destroy(string $id){
-        $employee = Employee::find($id);
-        Employee::destroy($id);
-        return Response::json( "Employee " . $employee->name . " and related Bank Account Records were Deleted");
+        try {
+            $employee = Employee::find($id);
+            $msg = "Employee " . $employee->name . " and related Bank Account Records were Deleted";
+            $employee->destroy($id);
```

Comment:
`$employee->destroy($id);` Do we need to pass the `$id` inside destroy builder?

Reply:
Yes we need to...or else im getting the few arguments error...it seems like the `delete() ` method doesn't require the id...but the destroy method does. Also destroy method removes data from the database without retrieving therefore it needs the primary key.

https://laravel.com/docs/11.x/eloquent#deleting-an-existing-model-by-its-primary-key

Reply:
You reference the Doc which says `Flight::destroy(1);` notice they haven't use `find(1)` builder instead directly used `destroy(1)`. please clarify the destroy process.

Reply:
Yeah okay i must use the delete() instead of destroy in a query builder ill make the changes...